numba_cuda/numba/cuda/cpython/cmathimpl.py

```diff
@@ -0,0 +1,566 @@
+"""
+Implement the cmath module functions.
+"""
+
+import cmath
+import math
+
+from numba.core.imputils import impl_ret_untracked
+from numba.core import types
+from numba.core.typing import signature
+from numba.cuda.cpython import mathimpl
+from numba.core.extending import overload
```

Comment:
I think the uses of overload should be modified to be

```python
@overload(func, target="cuda")
```

(i.e. make them CUDA target-specific)

so that we're not adding duplicate overloads for the generic target in Numba.